### PR TITLE
Member plugin

### PR DESCRIPTION
**Pull request by owaincuvelier**

### CODE REVIEW

File: load.php [hunk 1]
```diff
@@ -65,6 +65,7 @@ function get_config() {
 		'aws-ses-wp-mail' => true,
 		'tachyon'         => true,
 		'cavalcade'       => true,
+		'members'         => false,
```
Comment (joehoyle) [inline load.php]: can we call this `private` maybe? to be more descriptive

Comment (owaincuvelier) [reply load.php]: that just feels confusing to me from a developers perspective. But I'm happy to change it!

Comment (joehoyle) [reply load.php]: perhaps we can get a better description, `members` is not meant to be "Members only plugin", but really "make site private", the fact the plugin is called "members only" doesn't really matter, we want it to be a label that makes sense when a developer wants to make a site private. However, maybe `logged-in-only` (which we can default to false) would also work?

Comment (joehoyle) [reply load.php]: `members` isn't really WordPress user terminology, that's the main problem I think.

**Comment by rmccue** [comment]
Are we sure we want this plugin? This plugin is **not** the same as the one we use on [the HM internal sites](https://github.com/humanmade/Human-Made-website/blob/master/content/plugins-mu/hm-user-redirect/hm-user-redirect.php).

File: load.php (hunk 1 above)
Comment (owaincuvelier) [reply load.php]: Right, that makes sense to me. `logged-in-only` sounds fine for now.

**Comment by owaincuvelier** [comment]
@rmccue we want an easy way for developers to block access to staging sites while making it easy for clients to have access. This plugin seems to do that quite well.

**Comment by fklein-lu** [comment]
It seems kind of overkill to add this Members plugin just for that, especially since there's [Maintenance Mode](https://github.com/Automattic/maintenance-mode-wp) and similar options.

**Comment by joehoyle** [comment]
> Are we sure we want this plugin? This plugin is not the same as the one we use on the HM internal sites.

This is a different use case

**Pull request closed without merging**